src/utf8.cpp

```diff
@@ -243,6 +243,15 @@ size_t normalize_codepoint(uint32_t codepoint, int32_t *wbBuffer, size_t wbBuffe
         return 1;
     }
 
+    // Hidden ASCII range (plane 14) https://en.wikipedia.org/wiki/Tags_(Unicode_block)
+    // [U+E0000, U+E0019] (excluding U+E0001) has been deprecated and is unassigned
+    if (codepoint == 0xE0001 || (codepoint >= 0xE0020 && codepoint <= 0xE007F)) {
```

Comment:
From the article it says these characters are used (with the exception of U+E0001, which remains deprecated). The subtraction of 0xE0000 and their moving into the ASCII range, I don't see what meaning it has. There is no relationship between these codepoints and the ASCII codepoints other than some vague organization resemblance.